Windows: Link error LNK2001: undefined symbol Java_sun_security_mscapi, with JDK 17

<!--- Provide a brief summary of the issue -->
With GraalVM based on JDK 17, Substrate fails to link on Windows:

```
[SUB] hellofx.hellofxml.obj : error LNK2001: unresolved external symbol Java_sun_security_mscapi_CKeyPairGenerator_00024RSA_generateCKeyPair
[SUB] hellofx.hellofxml.obj : error LNK2001: unresolved external symbol Java_sun_security_mscapi_CKeyStore_loadKeysOrCertificateChains
[SUB] hellofx.hellofxml.obj : error LNK2001: unresolved external symbol Java_sun_security_mscapi_CKey_cleanUp
[SUB] hellofx.hellofxml.obj : error LNK2001: unresolved external symbol Java_sun_security_mscapi_CKey_getContainerName
[SUB] hellofx.hellofxml.obj : error LNK2001: unresolved external symbol Java_sun_security_mscapi_CKey_getKeyType
[SUB] hellofx.hellofxml.obj : error LNK2001: unresolved external symbol Java_sun_security_mscapi_CPublicKey_00024CRSAPublicKey_getExponent
[SUB] hellofx.hellofxml.obj : error LNK2001: unresolved external symbol Java_sun_security_mscapi_CPublicKey_00024CRSAPublicKey_getModulus
[SUB] hellofx.hellofxml.obj : error LNK2001: unresolved external symbol Java_sun_security_mscapi_CPublicKey_getPublicKeyBlob
[SUB] hellofx.hellofxml.obj : error LNK2001: unresolved external symbol Java_sun_security_mscapi_CRSACipher_encryptDecrypt
[SUB] hellofx.hellofxml.obj : error LNK2001: unresolved external symbol Java_sun_security_mscapi_CSignature_00024RSA_generatePublicKeyBlob
[SUB] hellofx.hellofxml.obj : error LNK2001: unresolved external symbol Java_sun_security_mscapi_CSignature_importECPublicKey
[SUB] hellofx.hellofxml.obj : error LNK2001: unresolved external symbol Java_sun_security_mscapi_CSignature_importPublicKey
[SUB] hellofx.hellofxml.obj : error LNK2001: unresolved external symbol Java_sun_security_mscapi_CSignature_signCngHash
[SUB] hellofx.hellofxml.obj : error LNK2001: unresolved external symbol Java_sun_security_mscapi_CSignature_signHash
[SUB] hellofx.hellofxml.obj : error LNK2001: unresolved external symbol Java_sun_security_mscapi_CSignature_verifyCngSignedHash
[SUB] hellofx.hellofxml.obj : error LNK2001: unresolved external symbol Java_sun_security_mscapi_CSignature_verifySignedHash
[SUB] hellofx.hellofxml.obj : error LNK2001: unresolved external symbol Java_sun_security_mscapi_PRNG_generateSeed
[SUB] hellofx.hellofxml.obj : error LNK2001: unresolved external symbol Java_sun_security_mscapi_PRNG_getContext
[SUB] hellofx.hellofxml.obj : error LNK2001: unresolved external symbol Java_sun_security_mscapi_PRNG_releaseContext
```

### Expected Behavior
<!--- Tell us what should happen -->

### Current Behavior
<!--- Tell us what happens instead of the expected behavior -->

### Steps to Reproduce
<!--- Steps to reproduce this bug -->

### Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->

--- a/src/main/java/com/gluonhq/substrate/target/WindowsTargetConfiguration.java
+++ b/src/main/java/com/gluonhq/substrate/target/WindowsTargetConfiguration.java
@@ -42,9 +42,9 @@ import java.util.stream.Stream;
 public class WindowsTargetConfiguration extends AbstractTargetConfiguration {
 
     private static final List<String> javaWindowsLibs = Arrays.asList(
-            "advapi32", "iphlpapi", "secur32", "userenv", "version", "ws2_32", "winhttp");
+            "advapi32", "iphlpapi", "secur32", "userenv", "version", "ws2_32", "winhttp", "ncrypt", "crypt32");
     private static final List<String> staticJavaLibs = Arrays.asList(
-            "j2pkcs11", "java", "net", "nio", "prefs", "fdlibm", "sunec", "zip");
+            "j2pkcs11", "java", "net", "nio", "prefs", "fdlibm", "sunec", "zip", "sunmscapi");
     private static final List<String> staticJvmLibs = Arrays.asList(
             "jvm", "libchelper");
 

